<ui version="4.0" >
 <class>SimSetupDialog</class>
 <widget class="QDialog" name="SimSetupDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Run Dynamics</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="toolTip" >
   <string>Run Dynamics Setup Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="whatsthis_btn" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>21</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_sim_btn" >
       <property name="text" >
        <string>Run Simulation</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="parms_grpbox" >
     <property name="title" >
      <string>Simulation parameters</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel5" >
            <property name="text" >
             <string>Total frames:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel2" >
            <property name="text" >
             <string>Steps per frame:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="textLabel3" >
            <property name="text" >
             <string>Temperature:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>4</number>
          </property>
          <item>
           <widget class="QSpinBox" name="totalFramesSpinBox" >
            <property name="toolTip" >
             <string>Total Frames value</string>
            </property>
            <property name="suffix" >
             <string> frames</string>
            </property>
            <property name="maximum" >
             <number>1000000</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="singleStep" >
             <number>15</number>
            </property>
            <property name="value" >
             <number>900</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="stepsPerFrameDoubleSpinBox" >
            <property name="suffix" >
             <string> femtoseconds</string>
            </property>
            <property name="decimals" >
             <number>2</number>
            </property>
            <property name="singleStep" >
             <double>0.100000000000000</double>
            </property>
            <property name="value" >
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="temperatureSpinBox" >
            <property name="toolTip" >
             <string>Temperature</string>
            </property>
            <property name="suffix" >
             <string> K</string>
            </property>
            <property name="maximum" >
             <number>99999</number>
            </property>
            <property name="value" >
             <number>300</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="potential_energy_checkbox" >
        <property name="text" >
         <string>Plot energy in tracefile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="watch_motion_groupbox" >
     <property name="title" >
      <string>Watch motion in real time</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="update_every_rbtn" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during the simulation.</string>
          </property>
          <property name="text" >
           <string>Update every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="update_number_spinbox" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during the simulation.</string>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="update_units_combobox" >
          <property name="toolTip" >
           <string>Specify how often to update the screen during the simulation.</string>
          </property>
          <item>
           <property name="text" >
            <string>frames</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>seconds</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>minutes</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>hours</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>71</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="update_asap_rbtn" >
        <property name="toolTip" >
         <string>Update every 2 seconds, or faster if it doesn't slow simulation by more than 20%</string>
        </property>
        <property name="text" >
         <string>Update as fast as possible</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="md_engine_groupbox" >
     <property name="title" >
      <string>Molecular Dynamics Engine</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <property name="spacing" >
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>4</number>
        </property>
        <item>
         <widget class="QComboBox" name="simulation_engine_combobox" >
          <property name="toolTip" >
           <string>Choose the simulation engine with which to minimize energy.</string>
          </property>
          <item>
           <property name="text" >
            <string>NanoDynamics-1 (Default)</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>GROMACS</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="electrostaticsForDnaDuringDynamics_checkBox" >
        <property name="text" >
         <string>Electrostatics for DNA reduced model</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>SimSetupDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
